### ISSUE TITLE
Titan 1.x support for Elasticesearch 2.x

### ISSUE THREAD

**Issue by zazzcode** [thinkaurelius/titan#1188]
Hi,
Can a future release of Titan 1.x be updated to support Elasticsearch 2.x.
The query syntax has changed from Elasticsearch 1.x to 2.x, so this may not be a trivial effort.

**Comment by dylanht** [comment]
I posted in the mailing list a while ago about getting the Titan branch that is staying current with TinkerPop3 to build against ElasticSearch development branch (3.0.0-SNAPSHOT) and Lucene 5.4 which that and possibly 2.x requires - I stopped short of actually getting a cluster fired up, as dealing with the new "JarHell" class in ES when I clearly am living in Jar hell was just too tedious, in theory it was going to work after I fixed my lazy CLASSPATH issues:

https://groups.google.com/forum/#!searchin/aureliusgraphs/ElasticSearch%7Csort:date/aureliusgraphs/Q6ccAz7dBZg/L8pyZwFLBwAJ

I then realized that someone had already done pretty much the same work but probably much better, and if you ask them they probably have more confidence as to why they did what they did - check out this PR:

https://github.com/thinkaurelius/titan/pull/1153

So in summary, as someone who got this to build and saw the Daemon kill itself when launching Titan + ES + Gremlin-Server with bin/titan.sh only because I put a lot of garbage in my classpath, I would say yes.
